test/passing/warning50.ml.ref

```diff
@@ -0,0 +1,6 @@
+ocamlformat: Cannot process "warning50.ml".
+  Please report this bug at https://github.com/ocaml-ppx/ocamlformat/issues.
+  BUG: misplaced documentation comments.
+File "warning50.ml", line 8, characters 8-56:
+Warning 50: unattached documentation comment (ignored)
+Hint: (Warning 50) disable this check using the `--no-comment-check` option
```

Comment:
I think that this hint will be useful. But I'm not sure whether the whole output is actionable.

When this happens, what should the user do? Should they report the bug, as prompted? It seems to me that ocamlformat is not designed to handle files that emit instances of warning 50. If that is correct, I suggest replacing it with something like that:

> This file contains a documentation comment `(** ... *)` that the OCaml compiler does not know how to attach to the AST. OCamlformat does not support these cases. You can find more information at: <https://github.com/ocaml-ppx/ocamlformat#overview>. If you'd like to disable this check and let ocamlformat make a choice (though it might not be consistent with the ocaml compilers and odoc), you can set the `--no-comment-check` option.